resources/eventing/charts/controller/templates/clusterrole.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ include "controller.fullname" . }}
+  labels: {{- include "controller.labels" . | nindent 4 }}
+rules:
+- apiGroups:
+  - ""
+  resources:
+  - services
+  verbs:
+  - list
+  - get
+  - watch
+- apiGroups:
+  - ""
+  resources:
+  - events
+  verbs:
+  - create
+  - patch
+- apiGroups:
+  - eventing.kyma-project.io
+  resources:
+  - subscriptions
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - eventing.kyma-project.io
+  resources:
+  - subscriptions/status
+  verbs:
+  - get
+  - patch
+  - update
+- apiGroups:
+  - eventing.kyma-project.io
+  resources:
+  - subscriptions
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - gateway.kyma-project.io
+  resources:
+  - apirules
+  verbs:
+  - get
+  - list
+  - watch
+  - patch
+  - update
+  - create
```

Comment:
What do you think about introduction of PodSecurityPolicy now? It will be required in 1.19 anyway

Reply:
The images are changed to using [gcr.io/distroless/static:nonroot](https://github.com/kyma-project/kyma/blob/master/components/eventing-controller/Dockerfile#L20). Hence PSPs are not relevant here.

Reply:
Great!